resources/puppetlabs/lein-ezbake/template/foss/ext/debian/ezbake.service.erb

```diff
@@ -12,6 +12,7 @@ Restart=on-failure
 StartLimitBurst=5
 #set default privileges to -rw-r-----
 UMask=027
+LimitNOFILE=${OPEN_FILE_LIMIT}
```

Comment:
Ah, ok. That seems to be consistent with [this note](https://lists.freedesktop.org/archives/systemd-devel/2015-September/034092.html).